lib/shuffle.js

```diff
@@ -113,9 +114,11 @@ function randomRange(min, max) {
 module.exports = function shuffle(array) {
```

Comment:
Since we return the array, can we not mutate the original one? (That'll require an end-to-end though, so that can wait for later)